src/client/components/Forms/ResetPassword.js

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+
+export default function ResetPassword({ onSubmit }) {
+  const [email, setEmail] = useState('');
+  const handleEmailInput = (e) => setEmail(e.target.value);
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+    await onSubmit({ email });
+  };
+  return (
+    <form onSubmit={handleSubmit}>
+      <label htmlFor="email">
+        Email:{' '}
+        <input
+          type="email"
+          name="email"
+          value={email}
+          onChange={handleEmailInput}
+          required
+        />
```

Comment:
Placeholders for all inputs would be nice

```suggestion
        <input
          type="email"
          name="email"
          value={email}
          onChange={handleEmailInput}
          required
          placeholder="Write your email"
        />
```